cmd/kindnetd/main.go

```diff
@@ -127,12 +174,54 @@ func makeNodesReconciler(cniConfigWriter *CNIConfigWriter, hostIP string) func(*
 		}
 
 		fmt.Printf("Handling node with IP: %s\n", nodeIP)
+		fmt.Printf("Node %v has CIDR %s \n", node.Name, podCIDR)
+		kindnetd.config.PodCIDRs[podCIDR] = nodeIP
```

Comment:
this is not really a reconcile and I don't like this separation of concerns, but functionally this _works_